airflow/models/taskinstance.py

```diff
@@ -1087,7 +1087,7 @@ def _run_raw_task(
             self.refresh_from_db(lock_for_update=True)
             self.state = State.SKIPPED
             self.log.info(
-                'Marking task as SKIPPED.'
+                'Marking task as SKIPPED. '
                 'dag_id=%s, task_id=%s, execution_date=%s, start_date=%s, end_date=%s',
```

Comment:
```suggestion
                '(dag_id=%s, task_id=%s, execution_date=%s, start_date=%s, end_date=%s)',
```
Just an idea to make it more "structure" but definitely not a blocker